src/sdkClient/clientAttributesDecoration.ts

```diff
@@ -19,25 +19,33 @@ export function clientAttributesDecoration<TClient extends SplitIO.IClient | Spl
   const clientTrack = client.track;
 
```

Comment:
in line 10, please remove the `any` as return type, so that TS automatically infer it for our UTs and the rest of the code.